client/html/layouts/basket/mini/product-body-default.html

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @copyright Copyright (c) Joyce Darimont, 2015
+ * @license LGPLv3, http://opensource.org/licenses/LGPL-3.0
+ */
+
+
+try
+{
+	$miniPriceValue = $this->productPriceItem->getValue();
+	$miniPriceCosts = $this->productPriceItem->getCosts();
+	$miniPriceCurrency = $this->translate( 'client/html/currency', $this->mainPriceItem->getCurrencyId() );
+}
+catch( Exception $e )
+{
+	$miniPriceValue = '0.00';
+	$miniPriceCurrency = '';
+}
+
+
+
+$products = $this->get( 'productItems', 0 );
```

Comment:
The last problem: You expect an array, so the second parameter must be "array()"